admin_guide/assembly_restoring-cluster.adoc

```diff
@@ -0,0 +1,66 @@
+[[admin-guide-restore]]
+= Restoring your cluster
+{product-author}
```

Comment:
This title confuses me a bit. The entire topic is about restoring a cluster, but there is a singular section on restoring a cluster. 
Maybe _Restore your {product-title} environment_?

Reply:
I'd agree. It might be more accurate to say "Restoring {product-title} components" or "Restoring parts of your {product-title} environment".

Reply:
"Restoring {product-title} components" it is. :)